src/components/Output/style.scss

```diff
@@ -15,12 +15,176 @@ Note: These styles are temporary. They will be replaced before going live.
 .output {
   @extend %fill;
 
+  --background-color: rgba(0,0,0,0.05);
+  background-color: #fff;
+  background-repeat: repeat;
+  background-image:
+    linear-gradient(45deg, var(--background-color) 25%, transparent 25%),
+    linear-gradient(-45deg, var(--background-color) 25%, transparent 25%),
+    linear-gradient(45deg, transparent 75%, var(--background-color) 75%),
+    linear-gradient(-45deg, transparent 75%, var(--background-color) 75%);
+  background-size: 20px 20px;
+  background-position: 0 0, 0 10px, 10px -10px, -10px 0px;
+  transition: background-color 300ms ease;
```

Comment:
This gonna be incredibly slow on big screens. If we want to animate this we should use a pseudo-element and animate `opacity` instead.

Reply:
It’s a solid 10fps on my 4k :D

Reply:
Yeah, since we're aiming to showcase the best of web performance, we should avoid animating with paints unless it can be justified.

For instance, pinch-zoom is smooth because it's all compositing (although it's main-thread blocked due to input).

two-up lays out & paints on the main thread, but since the innards are composited, it takes ~1ms. So it's justified to use clip-path rather than silly overflow:hidden tricks.

Reply:
- [ ] use composited opacity for background animation